components/DetailPullup.tsx

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import { StyleSheet, Image, FlatList } from "react-native";
+import { View, Text } from "./Themed";
+
+const item = {
+  title: "Blue Pants",
```

Comment:
I think we should stay consistent with double or single quotation marks in our code.  The convention for javascript/typescript is using single quotes.  Not sure if you should change this but I feel like we should discuss this in work session at some point so everything stays consistent

Reply:
Good point. Feel free to bring it up in work session!